src/Message/BitbucketImportCommand.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Message;
+
+class BitbucketImportCommand
+{
+    private string $username;
+
+    public function __construct(string $username)
+    {
+        $this->username = $username;
+    }
+    public function getOrgName(): string
+    {
+        return $this->username;
+    }

```

Comment:
This seems like duplication with GithubImportCommand any idea how to remove duplicated code there?

Reply:
Founded solution